scholia/network.py

```diff
@@ -8,6 +8,7 @@
 from collections import OrderedDict
 
 import requests
+from scholia.config import config
```

Comment:
Shouldn't this  be `from .config import config`

Reply:
I think this should be a relative import, i.e., `from .config import config` instead of `from scholia import config`